packages/ra-ui-materialui/src/layout/AppBar.js

```diff
@@ -61,6 +62,10 @@ const styles = theme => ({
     },
 });
 
+const styleHeadroom = {
```

Comment:
This style is always applied on the `appBar`, why not setting it on the `appBar` class directly ?

Reply:
Because `style` update inline style and I need to force the position, look [headroom/index.js](https://github.com/KyleAMathews/react-headroom/blob/master/src/index.js#L312)